Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2943,7 +2958,7 @@ extension BrowserViewController: IntroViewControllerDelegate {
 
     func introViewControllerDidRequestToLogin(_ introViewController: IntroViewController) {
         introViewController.dismiss(animated: true, completion: { () -> Void in
-            let fxaOptions = FxALaunchParams(view: "signup", email: nil, access_code: nil)
+            let fxaOptions = FxALaunchParams(query: nil)
```

Comment:
Yea it is only used when launching FxA viewcontroller via a deep link. Looks like I can declare a default value which would allow me to remove these statements.